cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/CassandraScheduler.java

```diff
@@ -240,196 +236,269 @@ public void error(final SchedulerDriver driver, final String message) {
 
     // ---------------------------- Helper methods ---------------------------------------------------------------------
 
-    /**
-     * @return boolean representing if the the offer was used
-     */
-    private boolean evaluateOffer(final SchedulerDriver driver, final Offer offer) {
+    private static boolean isExecutorTask(TaskID taskId) {
```

Comment:
We should avoid doing things like this and instead use the data model to express these things, either through a type or an enum.